.github/workflows/update-requirements.yml

```diff
@@ -94,10 +95,24 @@ jobs:
           name: requirements.txt
           path: requirements
 
+      - name: Commit requirements files
+        run: |
+          git config --local user.email "action@github.com"
+          git config --local user.name "GitHub Action"
+          git add .
```

Comment:
actually, I would be a bit more specific here and add only the `requirements/` folder.
other things might be added to the repo when the tests are running

Reply:
Which is why this step is executed in a separate job and only the artifact is downloaded (and thus added) to the repository. We can still be more explicit here if you like.

Reply:
You're right - in that case it's not mandatory... 
we could still add it to make the purpose explicit